nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,92 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+router.get("/", async (request, response) => {
+  try {
+    console.log(meals);
+    console.log("in /api/meals");
+  } catch (error) {
+    throw error;
+  }
+  //Get meals that has a price smaller than maxPrice
+  const maximumPrice = request.query.maxPrice;
+  const mealTitle = request.query.title;
+  const createdAfter = request.query.createdAfter;
+  const limit = request.query.limit;
+  let keysAreInValid = false;
+  //If the users writes a query parameter that is not supported
+  Object.keys(request.query).forEach(function (key) {
```

Comment:
This check could have been written more concisely, something like:
const parameterKeys = Object.keys(request.query).filter(...)
....